code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+import React, {useState, useEffect } from 'react'
+
+import MessageForm from './components/MessageForm'
+import MessageList from './components/MessageList'
+
+import { API_URL, LIKES_URL } from './reusable/urls'
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([])
+  const [messageNew, setMessageNew] = useState('')
+
+  useEffect(() => {
+    fetchMessageList()
+  }, [])
+  
+  const fetchMessageList = () => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(messages => setMessageList(messages))
+      .catch(err => console.error(err))
```

Comment:
Good you added catch error, I forgot this myself in my project. ⭐️

Reply:
Yes, but console outputs can't be present in the production code.